topic_folders/instructor_training/email_templates_trainers.md

```diff
@@ -127,7 +123,7 @@ I’m happy to say that you have passed your teaching demonstration! You demonst
 
 I’ve forwarded this information to our Core Team. If this was the last step in your Instructor Training checkout, you should hear from them within about a week. 
 
-Welcome to The Carpentries Instructor community!
+Looking forward to having you as a Carpentries Instructor!
```

Comment:
I have been trying to reduce words like "looking" where reasonable alternatives are less ableist. In this case I think the sentence can be deleted or left as-is, assuming Trainers will customize anyway.
```suggestion

```